### ISSUE TITLE
MIDIControl hangs for me as soon as it shoui

### ISSUE THREAD

**Issue by Snickbrack** [Etuldan/MidiControl#9]

**Comment by Snickbrack** [comment]
I have cloned the code and I was able to track it down to:

OBSControl.cs
Line: 116
`obs.Connect("ws://" + options.options.Ip, options.options.Password);`

This is the deadlock for me.

**Comment by Snickbrack** [comment]
I have also created custom firewall rules to allow all connections from the MidiControl-App. But no luck.

**Comment by Etuldan** [comment]
Which version of obs-websocket is installed on OBS ?
v4.9.1 is the max supported at the moment for MIDIControl.

**Comment by Snickbrack** [comment]
Yes I saw this and had the version 5 installed earlier and switched back to v4.9.1 and this worked then.

But... I got an error then saying `Input string was not in a correct format` for line 122 in OBSControl.cs

I had to manually comment out the lines 122-126 to make it fully work.

But this was not that big deal.

Great tool btw :)

**Comment by Etuldan** [comment]
Did you entered the correct credentials that you registered on OBS-websocket ?

**Comment by Snickbrack** [comment]
Yes and the connection was possible and if I comment out these lines then I can use it as it is meant to be. Just these lines are "bad"

**Comment by Etuldan** [comment]
These "lines" are responsible for the connection to OBS, so removing them will maybe solve the hangup issue, but it would be then impossible to MIDIControl to connect to OBS.
You may have 2 instances of MIDIControl opened, thus, the 2nd one hangup

**Comment by Snickbrack** [comment]
No, actually the Lines with the Version-Check are not necessary for me. And I would not understand it if they would be.

Why are you doing a Version-Check of OBS-Websocket? Just mention the version-to-use somewhere and done.

**Comment by Etuldan** [comment]
Could you retry with the latest version 2.0.0.1 ?
